### PR TITLE
express

### PR DESCRIPTION
**Pull request by AndresBallares**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+app.get("/:verb/:adjective/:noun", (request, response) => {
+    const { verb, adjective, noun } = request.params;
+    response.send("Congratulations on starting a new project called" + " " + verb + "-"  + adjective  + "-" +  noun  + "!");
+});
+
+// app.get("/bugs", (request, response) => {
+//     response.send("<h1>99 little bugs in the code</h1>")
+// })
+
+app.get("/", (request, response) => {
```
Comment (Angiepmunoz) [inline app.js:17]: this route should probably go before the route on line 8

File: app.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+})
+
+app.get("/pokemon/:indexOfArray", (request, response) => {
+    const { indexOfArray } = request.params;
+    if(indexOfArray > indexOfArray.length) {
+        response.send(`Sorry, no pokemon found at ${indexOfArray}`)
+    } else {
+        response.send(pokemon[indexOfArray])
+    }
+})
```
Comment (Angiepmunoz) [inline app.js:48]: not sure why this is failing because it looks correct

File: app.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+
+
```
Comment (Angiepmunoz) [inline app.js:51]: looks like you are missing a route

**Comment by AndresBallares** [comment]
Hello Angie,

I thank you for feedback, your time and assistance.  I will look into this and resubmit.


Best,


sent from Outlook<http://aka.ms/weboutlook>

Andres Ballares

JOYEROS INVESTMENT LLC

36 W 47th Street Suite 706

New York, NY 10036


Ph: (347) 671- 3181

________________________________

**Pull request closed without merging**